opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -2163,7 +2163,7 @@ void CpGrid::addLgrsUpdateLeafView(const std::vector<std::array<int,3>>& cells_p
                     this-> mark(1, element);
                     assignRefinedLevel[element.index()] = level+1; // shifted since starting grid is level 0, and refined grids levels are >= 1.
                     ++marked_elem_level_count;
-                    lgrs_with_at_least_one_active_cell[level] = marked_elem_level_count;
+                    active_cell_count_per_level[level] = marked_elem_level_count;
```

Comment:
Unfortunately, the name isn’t very clear. The variable active_cell_count_per_level[in certain level] stores the total number of active coarse cells that will be refined, with their child cells belonging to that specific level. I completely agree that it can be confusing, sorry for that!

Currently, we only support values of 0 and 1 for potential marks, which indicate 'do nothing to this element' and 'refine this element,' respectively. The variable marked_elem_level_count increases each time we encounter an element marked for refinement (line 2265 in the PR). At the end of the loops, this count represents the total number of marked elements to be refined, potentially across different level grids.

Reply:
Seems like `marked_elem_level_count` is never really used now. Please remove it.

Reply:
Done, thanks!